packages/wonder-blocks-popover/src/components/popover.tsx

```diff
@@ -56,6 +56,19 @@ type Props = AriaProps &
          * or clicking/tapping outside of it.
          */
         dismissEnabled?: boolean;
+
+        /**
+         * The type of content labelling this popover, if applicable.
+         * - title: References to the title of the popover.
+         * - content: References to the content of the popover.
+         * - all-content: References to both the title and content of the
+         *   popover.
+         *
+         * This is used to allow screen readers to announce the popover content
+         * when it is opened.
+         * Defaults to "title".
+         */
+        describedBy: "title" | "content" | "all-content";
```

Comment:
```suggestion
        describedBy: "title" | "content" | "title-and-content";
```

I think this is slightly clearer